src/liiteri/file_cleaner.clj

```diff
@@ -0,0 +1,43 @@
+(ns liiteri.file-cleaner
+  (:require [chime :as c]
+            [clojure.core.async :as a]
+            [clojure.java.jdbc :as jdbc]
+            [clj-time.core :as t]
+            [clj-time.periodic :as p]
+            [com.stuartsierra.component :as component]
+            [liiteri.db.file-metadata-store :as metadata-store]
+            [liiteri.files.file-store :as file-store]
+            [taoensso.timbre :as log]))
+
+(defn- clean-files [db storage-engine config]
+  (log/info "Cleaning files")
+  (jdbc/with-db-transaction [datasource db]
```

Comment:
Note: This is copy pasta from the virus checker and I don't understand what this macro does. Why isn't the `datasource` binding used?